packages/mdc-menu/README.md

```diff
@@ -227,22 +227,22 @@ Method Signature | Description
 `removeClassFromElementAtIndex(index: number, className: string) => void` | Removes the `className` class from the element at the `index` specified.
 `addAttributeToElementAtIndex(index: number, attr: string, value: string) => void` | Adds the `attr` attribute with value `value` to the element at the `index` specified.
 `removeAttributeFromElementAtIndex(index: number, attr: string) => void` | Removes the `attr` attribute from the element at the `index` specified.
-`elementContainsClass(element: HTMLElement, className: string) => boolean` | Returns true if the `element` contains the `className` class.
+`elementContainsClass(element: Element, className: string) => boolean` | Returns true if the `element` contains the `className` class.
 `closeSurface() => void` | Closes the menu surface.
-`getElementIndex(element: HTMLElement) => number` | Returns the `index` value of the `element`.
-`getParentElement(element: HTMLElement) => ?HTMLElement` | Returns the `.parentElement` element of the `element` provided.
-`getSelectedElementIndex(element: HTMLElement) => number` | Returns the `index` value of the element within the selection group provided, `element` that contains the `mdc-menu-item--selected` class.
+`getElementIndex(element: Element) => number` | Returns the `index` value of the `element`.
+`getParentElement(element: Element) => ?Element` | Returns the `.parentElement` element of the `element` provided.
```

Comment:
Also TS `lib.dom.d.ts` states it is an `HTMLElement`

```
/**
     * Returns the parent element.
     */
    readonly parentElement: HTMLElement | null;

```

Reply:
`lib.dom.d.ts` is wrong 😛 

* https://dom.spec.whatwg.org/#parent-element
* https://developer.mozilla.org/en-US/docs/Web/API/Node/parentElement